elastic-test/src/main/java/com/hazelcast/jet/tests/elastic/ElasticTest.java

```diff
@@ -0,0 +1,203 @@
+import com.hazelcast.jet.config.JobConfig;
+import com.hazelcast.jet.elastic.ElasticSinks;
+import com.hazelcast.jet.elastic.ElasticSourceBuilder;
+import com.hazelcast.jet.pipeline.BatchSource;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sink;
+import com.hazelcast.jet.pipeline.Sinks;
+import com.hazelcast.jet.pipeline.test.TestSources;
+import com.hazelcast.jet.tests.common.AbstractSoakTest;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+import org.apache.http.HttpHost;
+import org.elasticsearch.action.bulk.BulkRequest;
+import org.elasticsearch.action.delete.DeleteRequest;
+import org.elasticsearch.action.index.IndexRequest;
+import org.elasticsearch.action.search.SearchRequest;
+import org.elasticsearch.action.support.WriteRequest;
+import org.elasticsearch.client.RestClient;
+
+import static com.hazelcast.jet.tests.common.Util.sleepSeconds;
+import static java.util.stream.Collectors.toList;
+
+public class ElasticTest extends AbstractSoakTest {
+
+    private static final String SINK_LIST_NAME = ElasticTest.class.getSimpleName() + "_listSink";
+    private static final int DEFAULT_ITEM_COUNT = 10_000;
+    private static final int LOG_JOB_COUNT_THRESHOLD = 100;
+    private static final int SLEEP_BETWEEN_TABLE_READS_SECONDS = 5;
+
+    private String elasticIp;
+    private int itemCount;
+    private List<Integer> inputItems;
+
+    public static void main(String[] args) throws Exception {
+        new ElasticTest().run(args);
+    }
+
+    @Override
+    public void init(JetInstance client) throws SQLException {
+        elasticIp = property("elasticIp", "localhost");
+        itemCount = propertyInt("itemCount", DEFAULT_ITEM_COUNT);
+        inputItems = IntStream.range(0, itemCount).boxed().collect(toList());
+    }
+
+    @Override
+    public void test(JetInstance client, String name) throws Exception {
+        logger.info("Elastic ip: " + elasticIp);
+        clearSinkList(client);
+        int jobCounter = 0;
+        long begin = System.currentTimeMillis();
+        while (System.currentTimeMillis() - begin < durationInMillis) {
+            clearSinkList(client);
+
+            int indexCounter = jobCounter;
+
+            executeWriteToElasticPipeline(client, indexCounter);
+            executeReadFromElasticPipeline(client, indexCounter);
+            assertResults(client, jobCounter);
+
+            clearSinkList(client);
+
+            executeDeleteFromElasticPipeline(client, jobCounter);
+            executeReadFromElasticPipeline(client, indexCounter);
+
+            assertEmptyResults(client);
+
+            if (jobCounter % LOG_JOB_COUNT_THRESHOLD == 0) {
+                logger.info("Job count: " + jobCounter);
+            }
+
+            jobCounter++;
+            sleepSeconds(SLEEP_BETWEEN_TABLE_READS_SECONDS);
```

Comment:
we have 5 sec wait between jobs and we log for every 100 jobs. this makes it like every 8 minutes we log the counter. I think we should wither lower the wait time or the logging frequency (latter probably).